Packs/CarbonBlackEnterpriseEDR/ReleaseNotes/1_1_2.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### VMware Carbon Black Enterprise EDR
+**Breaking Change** the reputation commands **endpoint** was changed to return multiple entries (entry per indicator) instead of a single entry.
```

Comment:
This is not a breaking change - there was no change at all as far as the user is concerned.
```suggestion
Maintenance and stability enhancements.
```